database/store.py

```diff
@@ -0,0 +1,174 @@
+"""
+Data Access Object
+A data access object (DAO) is a pattern that provides an abstract interface
+to some type of database or other persistence mechanism.
+"""
+import datetime
+import uuid
+
+from database.initial_data import initial_application
+from database.initial_data import initial_application_store_state
+from dateutil import parser as date_parser
+from dateutil.tz import UTC
+from slugify import slugify
+from operator import itemgetter
+from distutils.util import strtobool
+from external_data.data import get_round
+
+
+class ApplicationDataAccessObject(object):
+    """
+    return all funds (replace the date object with a string to send)
+    """
+
+    def __init__(self):
+        self.applications: dict = initial_application_store_state
```

Comment:
another newbie python Q, should this be a private var, i.e: self._applications ... ? or maybe have a @property decorator. Please tell me if this is stupid

Reply:
I think good suggestion. Privatised.